cmake/Modules/FindVTune.cmake

```diff
@@ -26,19 +29,22 @@
 # ``VTune_FOUND``
 #   If false, do not try to use the VTune library.
 
-find_path(VTune_INCLUDE_DIR NAMES ittnotify.h HINTS ${VTUNE_PATH}/include)
+find_program(VTune_EXECUTABLE vtune HINTS ${VTune_PATH}/bin64 ${VTune_PATH}/bin32)
+get_filename_component(VTune_EXECUTABLE_DIR "${VTune_EXECUTABLE}" DIRECTORY)
+set(VTune_PATH ${VTune_EXECUTABLE_DIR}/..)
+find_path(VTune_INCLUDE_DIR NAMES ittnotify.h HINTS ${VTune_PATH}/include)
 mark_as_advanced(VTune_INCLUDE_DIR)
 
 if(NOT VTune_LIBRARY)
     if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-        find_library(VTune_LIBRARY NAMES ittnotify HINTS ${VTUNE_PATH}/lib64)
+        find_library(VTune_LIBRARY NAMES ittnotify HINTS ${VTune_PATH}/lib64)
     else()
-        find_library(VTune_LIBRARY NAMES ittnotify HINTS ${VTUNE_PATH}/lib32)
+        find_library(VTune_LIBRARY NAMES ittnotify HINTS ${VTune_PATH}/lib32)
     endif()
 endif()
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(VTune REQUIRED_VARS VTune_LIBRARY VTune_INCLUDE_DIR)
+find_package_handle_standard_args(VTune REQUIRED_VARS VTune_EXECUTABLE VTune_LIBRARY VTune_INCLUDE_DIR)
```

Comment:
does `VTune_EXECUTABLE` need to be required vars?

Reply:
It seems like Intel always bundles the executable with the library and header, so this seems cleanest to me. Without the executable being present, `VTune_PATH` would be incorrect, so I would need more complex logic for finding the library and header.